docs/content/en/docs/reference/configuration/tests.md

```diff
@@ -5,14 +5,128 @@ weight: 5
 aliases: ["/docs/reference/tests"]
 ---
 
-Vector enables you to unit test [transforms] in your processing topology
+Vector enables you to unit test [transforms] in your processing topology. The goal of unit tests in
+general is to improve the maintainability of configurations containing larger and more complex
+combinations of transforms.
 
-You can define unit tests in your Vector configuration file to cover a network of transforms within the topology. The intention of these tests is to improve the maintainability of configurations containing larger and more complex combinations of transforms.
+Unit tests in Vector work just like unit tests for standard software libraries:
+
+1. Provide a set of **inputs** to a transform (or a network of transforms)
```

Comment:
Network feels weird to me here as I'm not sure we use that to describe a set of transforms anywhere - do we want to reuse DAG/graph language here?

Reply:
This appears to be language held over from the old doc, but I think just `set` would be fine here.